java/test/jmri/jmrix/pi/RaspberryPiTurnoutManagerTest.java

```diff
@@ -13,15 +13,15 @@
 
 /**
  * <p>
- * Tests for RaspberryPiTurnoutManager
+ * Tests for RaspberryTurnoutManager
```

Comment:
The tested class is “RaspberryPiTurnoutManager” is there a reason you removed the “Pi”?

Reply:
Thanks for catching that.  Not sure why it happened, will fix.